src/services/ansibleSyntaxChecker.ts

```diff
@@ -0,0 +1,178 @@
+import * as child_process from 'child_process';
```

Comment:
Can rename this file to `ansiblePlaybook.ts` as it to extended to run other `ansible-playbook` sub-command in future if required

Reply:
The file has been renamed.